src/test/java/redis/clients/jedis/JedisSentinelPoolTest.java

```diff
@@ -18,9 +18,6 @@ public class JedisSentinelPoolTest {
 
   private static final String MASTER_NAME = "mymaster";
 
-  //protected static HostAndPort master = HostAndPorts.getRedisServers().get(2);
-  //protected static HostAndPort slave1 = HostAndPorts.getRedisServers().get(3);
-
   protected static final HostAndPort sentinel1 = HostAndPorts.getSentinelServers().get(1);
```

Comment:
Sentinels are managed the old way?

Reply:
I didn't touch sentinels and clusters because those tests need more work to support RE anyway.